contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -633,12 +663,16 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
      * the message information from `sentMessageInfo` and increments the reward redeemable by the specified relayer reward address.
      */
     function _markReceipt(
-        bytes32 destinationBlockchainID,
-        uint256 messageID,
+        bytes32 sourceBlockchainID_,
```

Comment:
nit: we should be consistent between `sourceBlockchainID` and `originBlockchainID` in our variable naming.

Reply:
I agree, and I think we should go with `sourceBlockchainID`, because because `subnet-evm` uses the "source" terminology for chain ID's. Interestingly, `subnet-evm` also uses `originSenderAddress`.

Deferring it to this issue though: https://github.com/ava-labs/teleporter/issues/189